README_Building.md

```diff
@@ -228,3 +463,49 @@ There's some more information on how to do this on the forum at http://forum.esp
 * Your board will now be flashed
 
 **Note:** if you want to you can change permissions so you don't need `sudo` by typing `sudo cp misc/45-espruino.rules /etc/udev/rules.d;sudo udevadm control --reload-rules` and then re-inserting the board.
+
+----
+
+Building with Bash on Ubuntu on Windows
+---------------------------------------
+
+* Install <https://msdn.microsoft.com/da-dk/commandline/wsl/install_guide>
+* After installation, obtain a bash prompt and use the following, but review the comments:
+
+```
+whoami # installation choice may mean i am root or non-root by default.
+
+sudo add-apt-repository ppa:team-gcc-arm-embedded/ppa # optional, but does no harm
+sudo apt-get update
+sudo apt-get upgrade
+
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash - # install node 
+
+sudo apt-get install -y \
+  build-essential lib32z1 lib32ncurses5 lib32bz2-1.0 lib32ncurses5 ia32-libs git 
+
+# Choose if to use the default ppa repository gcc-arm-embedded. Breaking edge.
+# sudo apt-get install -y gcc-arm-embedded
+# Alternativly one can use multiple installations and enable the version to be used with PATH and/or symlink
+mkdir -p /usr/local
+cd /usr/local
+wget https://launchpad.net/gcc-arm-embedded/5.0/5-2016-q3-update/+download/gcc-arm-none-eabi-5_4-2016q3-20160926-linux.tar.bz2
+tar xjf gcc-arm-none-eabi-5_4-2016q3-20160926-linux-tar-bz2
+wget https://developer.arm.com/-/media/Files/downloads/gnu-rm/6-2016q4/gcc-arm-none-eabi-6_2-2016q4-20161216-linux.tar.bz2
+tar xjf gcc-arm-none-eabi-6_2-2016q4-20161216-linux.tar.bz2
+export PATH=/usr/local/gcc-arm-none-eabi-5_4-2016q3-20160926-linux-tar-bz2/bin:$PATH
+
+# User choice for placement of source repos
+mkdir -p /mnt/c/source/repos/github/espruino
+cd /mnt/c/source/repos/github/espruino
+
+git clone https://github.com/espruino/EspruinoTools.git
+git clone https://github.com/espruino/Espruino.git
+git clone https://github.com/espruino/EspruinoDocs.git
+git clone https://github.com/espruino/EspruinoWebIDE.git
+
+cd Espruino
+make clean && make
+make clean && NRF52832DK=1 USE_FILESYSTEM=1 RELEASE=1 make
```

Comment:
Should probably be with puck.js and as an option with others: I do what the git master says